### DISCUSSION TITLE
contato

### DISCUSSION THREAD

**Discussion by AlexArantesBRx** [RamonSilva20/mapos#1760] [General]
gostaria de contato do criador tenho uma proposta para ele, e gostaria do contato do ramom silva alguém teria?

**Comment by Pr3d4dor** [comment]
@AlexArantesBRx o Ramon não participa mais ativamente do projeto, atualmente eu e o @willph somos os mantenedores do projeto.

**Comment by AlexArantesBRx** [comment]
Pode me passar algum contato pfv?

Enviado do Email<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows

De: Gianluca ***@***.***>
Enviado:terça-feira, 12 de abril de 2022 17:08
Para: ***@***.***>
***@***.***>; ***@***.***>
Assunto: Re: [RamonSilva20/mapos] contato (Discussion #1760)


@AlexArantesBRx<https://github.com/AlexArantesBRx> o Ramon não participa mais ativamente do projeto, atualmente eu e o @willph<https://github.com/willph> somos os mantenedores do projeto.
